operator/v1/types_network.go

```diff
@@ -321,6 +321,11 @@ type OVNKubernetesConfig struct {
 	// not using OVN.
 	// +optional
 	HybridOverlayConfig *HybridOverlayConfig `json:"hybridOverlayConfig,omitempty"`
+	// enableIPsec enables IPsec encryption on all pod-pod traffic within the cluster.
```

Comment:
```suggestion
	// enableIPsec enables IPsec encryption on all pod-to-pod traffic within the cluster.
```

Reply:
Also -> `EnableIPsec` to match the actual variable name.

Reply:
@dcbw @sttts Is there a convention here? I changed this to lower-case because sttts commented that "last nit: EnableIPsec in the godoc must be lower-case"

Reply:
@markdgray I guess I stand corrected? Aha, because the godoc is apparently for the *json* not the actual Go struct. Seems quite odd to me, but whatever.

Reply:
@dcbw Ok so I am making the assumption that the Go struct is capitalized but the corresponding JSON and godoc is lower-case. I will keep that change.